contracts/renderers/CrowdfundNFTRenderer.sol

```diff
@@ -29,6 +29,34 @@ contract CrowdfundNFTRenderer is RendererBase {
         IFont font
     ) RendererBase(globals, rendererStorage, font) {}
 
+    function contractURI() external view override returns (string memory) {
+        (bool isDarkMode, Color color) = getCustomizationChoices();
+        (string memory image, string memory banner) = getCollectionImageAndBanner(
+            color,
+            isDarkMode
+        );
+
+        return
+            string.concat(
+                "data:application/json;base64,",
+                Base64.encode(
+                    abi.encodePacked(
+                        '{"name":"',
+                        generateCollectionName(),
+                        '", "description":"',
+                        generateCollectionDescription(),
+                        '", "external_url":"',
```

Comment:
This is called `external_link` in the OpenSea docs [here](https://docs.opensea.io/docs/contract-level-metadata), but I would not be at all surprised if you are right and the docs are wrong, or if both are actually supported.